### PR TITLE
NEWEBODAC-34: Add create row functionality for all entities

### PR DESCRIPTION
**Pull request by kkaczmarczyk**
-added alert library
-added redirection after entity creation

### CODE REVIEW

File: src/main/webapp/js/actions/index.js [hunk 1]
```diff
@@ -125,8 +125,10 @@ export const updateEntity = (entityType, entity) => {
   };
 };
 
-export const createEntity = (entityType, entity) => {
+export const createEntity = (entityType, entity, callback) => {
   const request = apiClient.post(`${BASE_URL}/${entityType}`, entity);
+  request.then(() => callback());
```
Comment (kdondziak) [inline src/main/webapp/js/actions/index.js:130]: I think redux-thunk should be used here.

Comment (pmuchowski) [reply src/main/webapp/js/actions/index.js:130]: there is no need for redux-thunk here, it's just a single action

File: src/main/webapp/js/components/entities/entity-create.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            )}
+          />
+        </div>
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = (state, props) => ({
+  getEntityTypeOnly: getEntityTypeOnly(state, props),
```
Comment (pmuchowski) [inline src/main/webapp/js/components/entities/entity-create.jsx]: this is not used anywhere, remove this

File: src/main/webapp/js/selectors/index.js [hunk 3]
```diff
@@ -60,3 +60,6 @@ export const mapEntityToOptions = createSelector(
   getRelatedField,
   mapToOptions,
 );
+
+export const getEntityTypeOnly = (state, { entityType }) => (
+  entityType ? state.entity[entityType] : {});
```
Comment (pmuchowski) [inline src/main/webapp/js/selectors/index.js]: remove this

**Review by pmuchowski — CHANGES_REQUESTED** [review]

File: src/main/webapp/js/components/entities/entity-create.jsx (hunk 2 above)
Comment (kkaczmarczyk) [reply src/main/webapp/js/components/entities/entity-create.jsx]: fixed

File: src/main/webapp/js/selectors/index.js (hunk 3 above)
Comment (kkaczmarczyk) [reply src/main/webapp/js/selectors/index.js]: fixed

**Review by pmuchowski — APPROVED** [review]

**Pull request merged**